core/modules/comment/comment.install

```diff
@@ -354,13 +355,13 @@ function comment_update_1001() {
   foreach ($node_types as $node_type) {
     // Move the comment variables into each node type's config.
     $settings = array(
-      'comment_default' => update_variable_get('comment_' . $node_type->type, COMMENT_NODE_OPEN),
+      'comment_default' => update_variable_get('comment_' . $node_type->type, 2), // COMMENT_NODE_OPEN
```

Comment:
Just a minor coding standards suggestion, I'm not sure if you're using phpcs and what your attitude regarding these standards is (sorry for being picky).

phpcs nags: "Comments may not appear after statements."

Hint: The inline comments should be on their own line, start with a capital letter, end with a full-stop, exclamation mark, or question mark (and shouldn't be longer than 80 characters).

The comments explaining the "magic numbers" are per se a good idea!

Reply:
I found a lot of other similar comments in the code so figured it was ok. I looked at Backdrop coding standards and recall reading that for lists it's ok to put the comment at the end.

Reply:
At least that's how I read:

> If each line of a list needs a separate comment, the comments may be given on the same line and may be formatted to a uniform indent for readability.

Reply:
OK, then we should really update the phpcs coder sniffer variant to validate the stuff that's actually documented. :wink:
Feel free to mark this discussion as resolved (I can't).